jormungandr/src/rest/server/mod.rs

```diff
@@ -47,20 +47,14 @@ impl Server {
             let _ = sender.send(server_handler);
             if run_system {
                 actix_system.run();
+                process::exit(0);
```

Comment:
This is the most hidden way to stop the node.

Reply:
I can move it to main and trigger with MPSC channel